autorest/codegen/serializers/operation_groups_serializer.py

```diff
@@ -0,0 +1,68 @@
+# -------------------------------------------------------------------------
```

Comment:
@msyyc see above comment, I don't think adding all this code is necessary

Reply:
deleted